src/NuGet.Core/NuGet.Packaging.Extraction/NuGet.Packaging.Extraction.csproj

```diff
@@ -1,21 +1,67 @@
 ﻿<Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="ilmerge.props" />
```

Comment:
I checked out your branch and ran locally. The generated nupkg still has dependencies on the other packages.

Reply:
i've fixed it so that those are removed. the fix to remove frameworkassemblies didn't work...but i think that might even be good.

Reply:
here is the nuspec now:
```
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
  <metadata>
    <id>NuGet.Packaging.Extraction</id>
    <version>5.8.0-preview.3</version>
    <authors>Microsoft</authors>
    <owners>Microsoft</owners>
    <requireLicenseAcceptance>true</requireLicenseAcceptance>
    <license type="expression">Apache-2.0</license>
    <licenseUrl>https://licenses.nuget.org/Apache-2.0</licenseUrl>
    <projectUrl>https://aka.ms/nugetprj</projectUrl>
    <iconUrl>https://raw.githubusercontent.com/NuGet/Media/master/Images/MainLogo/256x256/nuget_256.png</iconUrl>
    <description>NuGet's understanding of packages. Reading nuspec, nupkgs and package signing.</description>
    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
    <tags>nuget</tags>
    <serviceable>true</serviceable>
    <repository type="git" url="https://github.com/NuGet/NuGet.Client" commit="bc805335d5921a86da2607ff1b095217e1b7ee0d" />
    <frameworkAssemblies>
      <frameworkAssembly assemblyName="System.IO.Compression" targetFramework=".NETFramework4.5" />
      <frameworkAssembly assemblyName="System.Security" targetFramework=".NETFramework4.5" />
      <frameworkAssembly assemblyName="System.Xml" targetFramework=".NETFramework4.5" />
      <frameworkAssembly assemblyName="System.Xml.Linq" targetFramework=".NETFramework4.5" />
    </frameworkAssemblies>
  </metadata>
</package>

```
the workaround that nuget.commandline.csproj used:
```
  <!-- Do nothing. This basically strips away the framework assemblies from the resulting nuspec.-->
  <Target Name ="_GetFrameworkAssemblyReferences" DependsOnTargets="ResolveReferences"/>
```
didn't work for this project.

1) i think it might be good to leave them here...because they do need to be referenced/available by projects using NPE.
2) happy to follow up if people disagree with #1

Reply:
> i've fixed it so that those are removed. the fix to remove frameworkassemblies didn't work...but i think that might even be good.

You probably need to put the target declaration after the common.targets import. 
Only suggestion I have is to try that. 

See https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj#L156
and see
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj#L152